### ISSUE TITLE
ARIA needed for components supporting multi selection.

### ISSUE THREAD

**Issue by benbcai** [cerner/terra-core#1205]
# Issue Description
The Button Group (and MultiSelectList and Table.MultiSelectableRows) supports multi selection.
These components should have an ARIA (e.g. [aria-multiselectable](https://www.w3.org/TR/wai-aria-1.1/#aria-multiselectable)) to indicate whether it is single or multi select.

## Issue Type
- [ ] New Feature
- [ ] Enhancement
- [ ] Bug
- [X] Other

**Comment by bjankord** [comment]
Maybe we could document this so consumers are aware if they are making the button group multi-selectable, they'll need to add this attribute.
